docs/config/README.md

```diff
@@ -495,6 +496,54 @@ The `buf` module shows the currently installed version of [Buf](https://buf.buil
 symbol = "🦬 "
 ```
 
+## C
+
+The `c` module shows some information about your C compiler. By default
+the module will be shown if the current directory contains a `.c` or `.h`
+file.
+
+### Options
+
+| Option              | Default                                                                     | Description                                                               |
+| ------------------- | --------------------------------------------------------------------------- | ------------------------------------------------------------------------- |
+| `format`            | `"via [$symbol$version(-$name )]($style)"`                                  | The format string for the module.                                         |
```

Comment:
I'm sorry to bother you with this again, but I noticed another issue with the default format string. If `$name` is missing, there won't be a space after the module. Please change it to `"via [$symbol($version(-$name) )]($style)"`.